CONTRIBUTING.md

```diff
@@ -30,6 +30,22 @@ works.
 
 This document will guide you through the contribution process.
 
+### Rules of thumb
+
+1. __Provide motivation for the change__  
+   Why will this change make the code better; does it fix a but, is it a new
+   feature, etc.
```

Comment:
Ok, good!
Also some PRs don't follow the rules and don't include the original commit messages.